beetsplug/kodinfo.py

```diff
@@ -0,0 +1,495 @@
+# -*- coding: utf-8 -*-
+# This file is part of beets.
+# Copyright 2017, Peace Lekalakala.
+# URL only text file by Sergio Soto, update by Pauli Kettunen.
+#
+# Permission is hereby granted, free of charge, to any person obtaining
+# a copy of this software and associated documentation files (the
+# "Software"), to deal in the Software without restriction, including
+# without limitation the rights to use, copy, modify, merge, publish,
+# distribute, sublicense, and/or sell copies of the Software, and to
+# permit persons to whom the Software is furnished to do so, subject to
+# the following conditions:
+#
+# The above copyright notice and this permission notice shall be
+# included in all copies or substantial portions of the Software.
+
+"""Creates Kodi nfo files in xml or url only text format.
+The nfos are created after importing album.
+
+Put something like the following in your config.yaml to configure:
+as per kodiupdate plugin
+    kodi:
+        host: localhost
+        port: 8080
+        user: user
+        pwd: secret
+        music_lib_name: music
+    audiodb:
+        key: secretkey
+"""
+from __future__ import absolute_import, division, print_function
+
+import base64
+import json
+import os
+import time
+import requests
+import urllib.request
+from urllib.request import Request, urlopen
+from uuid import UUID
+
+from beets import config
+import beets.library
+from beets.plugins import BeetsPlugin
+
+from pathlib import Path
+
+from lxml import etree as et
+
+import simplejson
```

Comment:
The changes have been effected. 
I guess I have to thank beets, for finally getting around to learning and most importantly understanding Python! Never touched python before trying to make this plugin. Back to learning!